layers/+vim/evil-better-jumper/README.org

```diff
@@ -19,7 +19,17 @@ locations.
 * Install
 To use this configuration layer, add it to your =~/.spacemacs=. You will need to
 add =evil-better-jumper= to the existing =dotspacemacs-configuration-layers=
-list in this file.
+list in this file. If =C-i= inserts an indent instead of
```

Comment:
C-i don't necessarily insert an indentation. It can binded to other functions.

Recommendation:

If =C-i= acts like =TAB= instead of ....

Reply:
Yes. Done.